### ISSUE TITLE
[REQ] Kotlin generator should propose to generate open classes

### ISSUE THREAD

**Issue by Oznup** [OpenAPITools/openapi-generator#16916]
### Is your feature request related to a problem? Please describe.

Hello, I'm using openapi-generator to generate kotlin classes at each build of the project.
These classes are in a library, and when used in other projects, we can't inherit from the generated classes because by default they're final.

## Describe the solution you'd like

Simply add an option (openClasses ?) that can be true or false.
If openClasses is true, the prefix "open" is added before the class declaration.

## Describe alternatives you've considered

Work directly on generated classes, but not possible in my context.
Create classes that contains the generated class, but that makes the model really ugly.

## Additional context

Nothing more to add.

**Comment by wing328** [comment]
> Simply add an option (openClasses ?) that can be true or false.

Yes, please file a PR with the suggested enhancement and we will start from there.
